File: Dev/JavaInterface/CarsInterface/src/structures/taxi/LocVertex.java
package structures.taxi;

public class LocVertex {

	private String area;
	private String location;
	private String locationType;
	private int progression;
	
	// Default constructor
	public LocVertex() {
		area="";
		location="";
		locationType="";
		progression=1;
	}
	
	//******************* GETTERS *****************//
	
	public String getArea() {
		return area;
	}
	
	public String getLocation() {
		return location;
	}
	
	public void setArea(String area) {
		this.area = area;
	}
	
	public String getLocationType() {
		return locationType;
	}
	
	public int getProgression() {
		return progression;
	}
	
	//***************** SETTERS *****************//
	
	public void setLocation(String location) {
		this.location = location;
	}
	
	public void setLocationType(String locationType) {
		this.locationType = locationType;
	}
	
	public void setProgression(int progression) {
		this.progression = progression;
	}
	
	
	@Override
	public String toString() {
		return "LocVertex [area=" + area + ", location=" + location
				+ ", locationType=" + locationType + ", progression="
				+ progression + "]";
	}
}

File: Dev/PythonServer/structure/cab/locvertex.py
class LocVertex:
    def __init__(self):
        self.area = ''
        self.location = ''
        self.locationType = ''
        self.progression = 1

    def parsing(self,value):
#        print 'LocVertex parsing'
 #       print value
        if 'area' in value:
            self.area = value['area']
        if 'location' in value:
            self.location = value['location']
        if 'locationType' in value:
            self.locationType = value['locationType']
        if 'progression' in value:
            self.progression = value['progression']

    def __eq__(self,other):
        test=True
        if self.area != other.area:
            test=False
        if self.location != other.location:
            test=False
        if self.locationType != other.locationType:
            test=False
        if self.progression != other.progression:
            test=False
        return test
